tests/travis/script.sh

```diff
@@ -11,5 +11,8 @@
 
 cd "$(dirname "$0")"; source _includes.sh
 
+# Exit early if no fixture exists.
```

Comment:
Why do we have our own script for this? Can't we simply use ORCA's?

Reply:
We may actually be able to use Orca's. I'm going to try that and see if I can eliminate this.